tests/scattering1d/test_filters_scattering1d.py

```diff
@@ -1,7 +1,7 @@
 """
 Testing all functions in filters_bank
 """
-from kymatio.scattering1d.filter_bank import (adaptive_choice_P, periodize_filter_fourier, get_normalizing_factor,
+from kymatio.scattering1d.filter_bank import (scattering_filter_factory, adaptive_choice_P, periodize_filter_fourier, get_normalizing_factor,
```

Comment:
Why was this not here before? @lostanlen @janden @AndreuxMath

Reply:
i don't know, i don't think it was deliberate